XCUITests/AuthenticationTest.swift

```diff
@@ -60,6 +60,7 @@ class AuthenticationTest: BaseTestCase {
     }
     
     fileprivate func enablePasscode(_ passCode: String) {
+        navigator.goto(NewTabMenu)
```

Comment:
I saw in BB that AuthenticationTest failed and locally with the changes in FxScreenGraph failed too. Adding this line here the tests work.
Not very clear to me why adding the NewPrivateTab to TabTray scene makes these tests to fail. It's that cannot go directly to Settings because instead of trying to opening it from the new tab, it tries to open it from TabTrayMenu. Need to understand why...